frontend/src/views/Landing.tsx

```diff
@@ -1,21 +1,213 @@
+                <CardHeader pad="medium">
+                  Very Important Event
+                </CardHeader>
+                <CardBody pad="medium">
+                  This is a summary of the very important event and why you 
+                  should totally go.
+                </CardBody>
+                <CardFooter pad="medium">
+                  <div>
+                    <p>Date: 1/1/2021</p>
+                    <p>Time: 1AM</p>
+                    <p>Place: under the button</p>
+                  </div>
+                </CardFooter>
+              </Card>
+              <Card height="medium" width="medium">
+                <CardHeader pad="medium">
+                  Very Important Event
+                </CardHeader>
+                <CardBody pad="medium">
+                  This is a summary of the very important event and why you 
+                  should totally go.
+                </CardBody>
+                <CardFooter pad="medium">
+                  <div>
+                    <p>Date: 1/1/2021</p>
+                    <p>Time: 1AM</p>
+                    <p>Place: under the button</p>
+                  </div>
+                </CardFooter>
+              </Card>
+              <Card height="medium" width="medium">
+                <CardHeader pad="medium">
+                  Very Important Event
+                </CardHeader>
+                <CardBody pad="medium">
+                  This is a summary of the very important event and why you 
+                  should totally go.
+                </CardBody>
+                <CardFooter pad="medium">
+                  <div>
+                    <p>Date: 1/1/2021</p>
+                    <p>Time: 1AM</p>
+                    <p>Place: under the button</p>
+                  </div>
+                </CardFooter>
+              </Card>
+            </Grid>
+          </Carousel>
+        </Section>
+        <AdditionalMargin>
+          <Section>
+            <LargeText style={{ display: "flex", justifyContent: "center", alignItems: "center" }}>
+              What We're Working On
+            </LargeText>
+            <Card pad="medium">
+              <CardHeader pad="medium">
+                Cool Project Name
+              </CardHeader>
+              <CardBody pad="medium">
+                This is what the project is about. It is really cool stuff. 
+                Everyone will read every single word of this description
+                because it's so captivating. Wow look at all this information.
+                This project is going so well. I am running out of words to put
+                in order to adequately fill this space. These are more words that
+                I am including because we will probably write a lot more about
+                this project because it's just that cool.
+              </CardBody>
+              <CardFooter style={{ display: "flex", justifyContent: "right", alignItems: "right" }}>

```

Comment:
Since you use this more than once (just below), I'd recommend using styled components to re-use this styling, instead of doing inlined styling!